ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -180,6 +180,19 @@
          (acl/verify-subscription-management-permission
            context :update :provider-object provider-id))))))
 
+(defn check-valid-user
+  "Raise error if the user provided does not exist"
+  [context user-id]
+  (when (string/blank? user-id)
+    (errors/throw-service-error
+     :bad-request
+     "INGEST FAILED - No ID was provided. Please provide a SubscriberId or pass in a valid token."))
```

Comment:
For clarity, instead of `INGEST FAILED`, I would prefer `Subscription creation failed - No ID...`